app/res/drawable/update_download.xml

```diff
@@ -5,8 +5,8 @@
         android:viewportHeight="232.5">
     <path
         android:pathData="M150,156.3l37.5,-56.3l-18.8,0l0,-56.2l-37.4,0l0,56.2l-18.8,0z"
-        android:fillColor="#D6EACC"/>
+        android:fillColor="#C1DFF5"/>
```

Comment:
@phillipm ready to pull after this super minor change. Can you ping me back on it when you make it so I can execute?